### DISCUSSION TITLE
Ian Wold | 90% of my Homepage was Useless

### DISCUSSION THREAD

**Discussion by giscus** [IanWold/ianwold.github.io#10] [Posts]
# Ian Wold | 90% of my Homepage was Useless

https://ian.wold.guru/Posts/90_of_my_homepage_was_useless.html

**Comment by IanWold** [comment]
One note to put here (I'm not sure it belongs on the article itself) is that my first blog post was [10 years and 1 day](https://ian.wold.guru/Posts/monokai_gray.html) before this one. I blogged very sporadically back then, but I'm glad I'm writing more regularly now, and just as well that it's been 10 years since I started!
